Dockerfile

```diff
@@ -85,7 +81,16 @@ COPY build/configuration-as-code/target/configuration-as-code.hpi /usr/share/jen
 COPY jenkins-configuration.yaml /usr/share/jenkins/ref/jenkins.yaml
 ENV CASC_JENKINS_CONFIG=$JENKINS_HOME/jenkins.yaml
 
+RUN chown -R $user:$group $EVERGREEN_HOME
+
+# Jenkins directory is a volume, so configuration and build history
+# can be persisted and survive image upgrades
+VOLUME ${EVERGREEN_HOME}
+
 # Ensure the supervisord configuration is copied and executed by default such
 # that the Jenkins and evergreen-client processes both execute properly
 COPY supervisord.conf /etc/supervisor/conf.d/supervisord.conf
 CMD /usr/bin/supervisord -c /etc/supervisor/conf.d/supervisord.conf
+
+WORKDIR $EVERGREEN_HOME
+USER jenkins
```

Comment:
I thought this should be `$user` rather than the constant string `jenkins`?

Reply:
Absolutely, good catch. I had that locally at some point when debugging the chown/volume issue, and forgot to re-adjust. Fixing